imports/ui/components/Card.jsx

```diff
@@ -1,14 +1,102 @@
 import React, { Component } from "react";
-import CardComponent from "../components/CardComponent";
+import { withStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import CardMedia from "@material-ui/core/CardMedia";
+import IconButton from "@material-ui/core/IconButton";
+import Typography from "@material-ui/core/Typography";
+import ThumbDownIcon from "@material-ui/icons/ThumbDown";
+import ThumbUpIcon from "@material-ui/icons/ThumbUp";
+import FavoriteIcon from "@material-ui/icons/FavoriteBorder";
+import FavoriteIconFilled from "@material-ui/icons/Favorite";
+
+class CardComponent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: this.props.post,
+      isFav: false
+    };
+  }
+
+  onFavPressed = () => {
+    this.setState({ isFav: !this.state.isFav });
+  };
 
-class Card extends Component {
   render() {
+    const { classes } = this.props;
     return (
       <div>
-        <CardComponent />
+        <Card className={classes.card}>
+          <CardMedia
+            className={classes.cover}
+            image="/static/images/cards/live-from-space.jpg"
+            title="Live from space album cover"
+          />
```

Comment:
This was already here already. So I'm not touching it to have it serve as a placeholder for our photos later.